omod/src/main/java/org/openmrs/module/registrationapp/converter/RegistrationSummaryExtensionsGenerator.java

```diff
@@ -89,9 +89,8 @@ public static boolean isRegAppConfig(AppDescriptor app) {
 	 * Generates a comprehensive registration summary ID from an app ID and section ID.
 	 */
 	public static String generateRegSummaryId(String regAppId, String sectionId) {
-		String regSummaryId = regAppId.substring(0, regAppId.indexOf("registerPatient")) 
+		return (regAppId.substring(0, regAppId.indexOf("registerPatient"))
 								+ "summary." 
```

Comment:
```suggestion
		return (regAppId.substring(0, regAppId.indexOf("registerPatient")) + "summary." + sectionId);
```
Why not in one line?

Ref: https://wiki.openmrs.org/display/docs/Java+Conventions#JavaConventions-Others
> Long lines. Lines should not exceed 125 characters.